Sources/StatsdClient/Documentation.docc/index.md

```diff
@@ -0,0 +1,49 @@
+# ``StatsdClient``
```

Comment:
I saw some of the NIO PRs are use `Docs.docc/index.md` and here and in distributed actors we use `Documentation.docc/index.md`. no strong opinion but I would prefer to get them consistent. which one do we prefer

cc @Lukasa @weissi @yim-lee @ktoso?

Reply:
It's my PRs that are not being consistent so let's stick with `Docs.docc`.

Reply:
Let's go with `Docs` I'll adjust others

Reply:
We have `Docs.docc` actually https://github.com/apple/swift-distributed-actors/tree/main/Sources/DistributedActors/Docs.docc :) al let's stick to that everywhere 👍